src/index.ts

```diff
@@ -184,3 +184,4 @@ export { default as WarningOutlineIcon } from './General/Icon/components/Warning
 export { default as WarningSolidIcon } from './General/Icon/components/WarningSolidIcon';
 export { default as WhatsappIcon } from './General/Icon/components/WhatsappIcon';
 export { default as YoutubeIcon } from './General/Icon/components/YoutubeIcon';
+export { default as RssIcon } from './General/Icon/components/RssIcon';
```

Comment:
This list of exports should be sorted alphabetically ;)

Reply:
Fixed!